### ISSUE TITLE
"AttributeError: 'file' object has no attribute 'buffer"

### ISSUE THREAD

**Issue by jchmielarz** [Intoli/exodus#12]
I run into the following issue while trying to use exodus.
Any tips?

Btw. very cool idea, many thanks, keep up the good work.

Command output:

```me@VirtualBox:~/Projects/cquery/build$ exodus ./cquery | ssh -T me@server
me@server's password: 
Traceback (most recent call last):
  File "/home/kuba/.local/bin/exodus", line 11, in <module>
    sys.exit(main())
  File "/home/kuba/.local/lib/python2.7/site-packages/exodus_bundler/cli.py", line 113, in main
    create_bundle(**args)
  File "/home/kuba/.local/lib/python2.7/site-packages/exodus_bundler/bundling.py", line 47, in create_bundle
    output_file = sys.stdout.buffer
AttributeError: 'file' object has no attribute 'buffer'
```

- System: Xubuntu 17.10
- Python 2.7
- Exodus v1.1.3
- "echo $LD_LIBRARY_PATH" is empty
- xfce4-terminal

**Comment by sangaline** [comment]
Thanks! This was caused by a minor Python 2 compatibility issue that was fixed in #11. If you update to v1.1.4, then it should work (`pip install --upgrade exodus-bundler`). Let me know if you're still experiencing problems and I'll reopen.

**Comment by jchmielarz** [comment]
Works very fine now! Pure magic :)
